qutebrowser/config/configdata.py

```diff
@@ -499,6 +499,11 @@ def data(readonly=False):
              SettingValue(typ.Bool(), 'true'),
              "Whether to show favicons in the tab bar."),
 
+            ('tabbar-size',
+             SettingValue(typ.Int(minval=8), '12'),
+             "The height of the tabbar in pixels."
```

Comment:
As these two strings simply get concatenated for the docs, there's a space missing after the dot here.